doc/dynakube-api-ref.md

```diff
@@ -0,0 +1,99 @@
+DynaKube is the Schema for the DynaKube API
+
+## DynaKube is the Schema for the DynaKube API
```

Comment:
duplicate + does not say much 

`DynaKube Schema` would work fine

Reply:
fixed in 8eca23cd